koji-setup/mash.sh

```diff
@@ -5,10 +5,11 @@
 set -e
 . /etc/profile.d/proxy.sh || :
 
+KOJI_TAG="${KOJI_TAG:-"$1"}"
 BUILD_ARCH="${BUILD_ARCH:-x86_64}"
 KOJI_DIR="${KOJI_DIR:-/srv/koji}"
 MASH_DIR="${MASH_DIR:-/srv/mash}"
-MASH_TRACKER_FILE="$MASH_DIR"/latest-mash-build
+MASH_TRACKER_FILE="$MASH_DIR"/latest-${KOJI_TAG}-build
```

Comment:
If I am understanding the intent of the PR correctly, I think we want
```
MASH_DIR="${MASH_DIR:-"/srv/mash/$KOJI_TAG"}" 
```
Then we wouldn't modify the tracker file name, but rather, self-contain a similar structure to every mash in the name of a folder that varies by tag.